pwa-devdocs/src/tutorials/cloud-deploy/index.md

```diff
@@ -2,238 +2,198 @@
+Depending on your access permissions for the default environment, this directory may appear empty.
 
-Run the following `composer` CLI commands to install the sample data modules into Magento:
+Navigate to the project directory and use the Magento Cloud CLI to list the environments for this project.
 
 ```sh
-composer require magento/sample-data-media-venia:dev-master
+magento-cloud environment:list
 ```
 
-```sh
-composer require magento/module-catalog-sample-data-venia:dev-master
-```
+This command displays a table of environments which lists their ID, Title, and Status.
 
 ```sh
-composer require magento/module-configurable-sample-data-venia:dev-master
+Your environments are:
++----------------------+----------------------+-------------+
+| ID                   | Title                | Status      |
++----------------------+----------------------+-------------+
+| production*          | Production           | Active      |
+| staging              | Staging              | Active      |
+| myStorefront-develop | myStorefront-develop | In progress |
+|    myStorefront-cicd | myStorefront-cicd    | Active      |
++----------------------+----------------------+-------------+
+* - Indicates the current environment
 ```
 
-```sh
-composer require magento/module-customer-sample-data-venia:dev-master
-```
+Use the Magento Cloud CLI to checkout the environment where you want to add your storefront code, such as the `staging` environment.
 
 ```sh
-composer require magento/module-tax-sample-data-venia:dev-master
+magento-cloud checkout staging
 ```
 
-```sh
-composer require magento/module-sales-sample-data-venia:dev-master
-```
+## Add required Magento extensions
 
+Magento Cloud does not support node processes, so you cannot use UPWARD-JS to serve your storefront project.
+You must use the [magento2-upward-connector][] module with [UPWARD-PHP][] to deploy your storefront in Magento Cloud.
 
-These commands modify the `composer.json` file and adds the sample data modules to the `require` section:
+Use the `composer` CLI command to add the magento2-upward-connector module to the Magento installation:
 
-```json
-"magento/module-catalog-sample-data-venia": "dev-master",
-"magento/module-configurable-sample-data-venia": "dev-master",
-"magento/module-customer-sample-data-venia": "dev-master",
-"magento/module-tax-sample-data-venia": "dev-master",
-"magento/module-sales-sample-data-venia": "dev-master",
-"magento/sample-data-media-venia": "dev-master",
+```sh
+composer require magento/module-upward-connector
 ```
 
-## Install dependencies
-
-### Composer dependencies
-
-_Optional_: If you manually modified the `composer.json` file to make the changes described in the previous steps, update your `composer.lock` with these new dependencies:
+This command modifies the `composer.json` file and adds the package entry to the `require` section of the `composer.json` file.
 
-```sh
-composer update
+```text
+"magento/module-upward-connector": "^1.0.1"
```

Comment:
Latest version is `1.1.2`, not sure if we should reference most up to date versions compatible with `2.4.x` release line. I know the `^` would still install latest minor and we don't version these docs, so just a call out.